### PR TITLE
candy machine bump anchor version

### PR DESCRIPTION
**Pull request by stranzhay**
round 2 for real this time. bumping anchor because forced to by the library/creator

### CODE REVIEW

**Review by blockiosaurus — COMMENTED** [review]
You can `return !err(CandyError::"error")` with the new Anchor and get rid of the `Err` && `.into()` calls to make things a bit cleaner. Not a dealbreaker but it would be good to be consistent across all products.

**Comment by samuelvanderwaal** [comment]
> 

Can you clarify what you mean here? Is there a type in this? `return !err(CandyError::"error")`

**Comment by blockiosaurus** [comment]
From Auction House
`return err!(ErrorCode::ExpectedSolAccount);`
Seems to be the new Anchor style.

File: candy-machine/program/src/lib.rs [hunk 1]
```diff
@@ -125,7 +124,7 @@ pub mod candy_machine {
                     &payer.key(),
                     &gatekeeper.gatekeeper_network,
                     None,
-                )?;
+                );
```
Comment (stegaBOB) [inline candy-machine/program/src/lib.rs:127]:
```suggestion
                )?;
```

Need to add that `?` back in.

**Comment by stranzhay** [comment]
can someone please approve and merge this? its a major blocker for the release of sugar on Friday

**Comment by stegaBOB** [comment]
Update on this. Converting to a draft PR for now. I will be consuming these changes into my upcoming candy machine v2 struct changes

**Pull request closed without merging**